src/models/generation/thermal_generation.jl

```diff
@@ -56,7 +56,7 @@ TechThermal(; realpower = 0.0,
           realpowerlimits = @NT(min = 0.0, max = 0.0),
           reactivepower = nothing,
           reactivepowerlimits = nothing,
-          ramplimits = @NT(up=0.0, down=0.0),
+          ramplimits = @NT(up=10.0, down=10.0),
```

Comment:
The default should be nothing, I still don't get why are you adding values here. The whole point is to make it work when there are not ramping limits as a nothing.